ModularTegustation/tegu_items/prosthetics/chainsword.dm

```diff
@@ -0,0 +1,54 @@
+/obj/item/organ/cyberimp/arm/chainsword
+	name = "chainsword implant"
+	desc = "A chainsword that can be retracted into your arm.."
+	contents = newlist(/obj/item/ego_weapon/city/handchainsword)
+	syndicate_implant = TRUE
+	//Doesn't need a nerf, the gun's kinda shit lmao
+
+/obj/item/organ/cyberimp/arm/chainsword/l
+	zone = BODY_ZONE_L_ARM
+	syndicate_implant = TRUE
```

Comment:
Already defined by /obj/item/organ/cyberimp/arm/chainsword, no need to re-define
```suggestion
```